go.mod

```diff
@@ -405,7 +405,7 @@ require (
 	github.com/jmhodges/clock v1.2.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/josharian/native v1.1.0 // indirect
-	github.com/jpillora/backoff v1.0.0 // indirect
+	github.com/jpillora/backoff v1.0.0
```

Comment:
Can you move that dependency to the first `required` block and run `go mod tidy`?

Reply:
Done.